EnvDTE.Host/EnvDTE.Host.csproj

```diff
@@ -5,6 +5,8 @@
         <AssemblyName>JetBrains.EnvDTE.Host</AssemblyName>
         <RootNamespace>JetBrains.EnvDTE.Host</RootNamespace>
         <IsPackable>false</IsPackable>
+        <SignAssembly>true</SignAssembly>
+        <AssemblyOriginatorKeyFile>..\Subplatform.snk</AssemblyOriginatorKeyFile>
```

Comment:
Won't this break the open-source build, folks? I can't see `Subplatform.snk` in this repo.

Reply:
As I can see, it was added in this Pull Request

Reply:
Ah, sorry, my bad. That's correct.